documentation/markup.rst

```diff
@@ -1118,22 +1144,25 @@ units as well as normal text:
 
    There is one required argument: the version from which the feature is
    deprecated.
+   Similarly to ``versionadded``, you should use the word ``next`` to indicate
+   the API will be first deprecated in the upcoming release.
 
    Example::
 
-      .. deprecated:: 3.8
+      .. deprecated:: next
 
 .. describe:: deprecated-removed
 
    Like ``deprecated``, but it also indicates in which version the feature is
    removed.
 
    There are two required arguments: the version from which the feature is
-   deprecated, and the version in which the feature is removed.
+   deprecated (usually ``next``), and the version in which the feature
+   is removed, which must be a specific version number (*not* ``next``).
```

Comment:
Does the tool that convert `next` check this?

Reply:
Not explicitly, but it will fail.
For a check with a nice error message, could you review https://github.com/python/cpython/pull/124623?

It's not a big priority, since `next` doesn't really make sense here.